src/plugins/LADTable/components/LADRow.vue

```diff
@@ -89,43 +94,80 @@ export default {
             .telemetry
             .subscribe(this.domainObject, this.updateValues);
 
-        this.openmct
-            .telemetry
-            .request(this.domainObject, {strategy: 'latest'})
-            .then((array) => this.updateValues(array[array.length - 1]));
+        this.requestHistory();
     },
     destroyed() {
         this.stopWatchingMutation();
         this.unsubscribe();
-        this.openmct.off('timeSystem', this.updateTimeSystem);
+        this.openmct.time.off('timeSystem', this.updateTimeSystem);
+        this.openmct.time.off('bounds', this.updateBounds);
     },
     methods: {
         updateValues(datum) {
-            this.timestamp = this.formats[this.timestampKey].format(datum);
-            this.value = this.formats[this.valueKey].format(datum);
-
-            var limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+            let newTimestamp = this.formats[this.timestampKey].parse(datum),
+                shouldUpdate = false,
+                limit;
+
+            if(this.inBounds(newTimestamp)) {
+
+                // if timestamp is set, need tocheck, else update
+                if(this.timestamp !== '---') {
+
+                    // if existing is in bounds, need to check, if not update
+                    if(this.inBounds(this.timestamp)) {
+
+                        // race condition check
+                        if(newTimestamp >= this.timestamp) {
+                            shouldUpdate = true;
+                        }
+                    } else {
+                        shouldUpdate = true;
+                    }
+                } else {
+                    shouldUpdate = true;
+                }
+
+                if(shouldUpdate) {
+                    this.timestamp = this.formats[this.timestampKey].parse(datum);
+                    this.value = this.formats[this.valueKey].format(datum);
+                    limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+
+                    if (limit) {
+                        this.valueClass = limit.cssClass;
+                    } else {
+                        this.valueClass = '';
+                    }
+                }
 
-            if (limit) {
-                this.valueClass = limit.cssClass;
-            } else {
-                this.valueClass = '';
             }
         },
+        requestHistory() {
+            this.openmct
+                .telemetry
+                .request(this.domainObject, {
+                    start: this.bounds.start,
+                    end: this.bounds.end,
```

Comment:
Unrelated to this change, but it came up recently as an issue so let's fix it while we're in here. We need to add a `size:1` to the request options. Nothing else needs to be done. The issue that was raised by a user is that the parameters sent with our 'latest' queries are inconsistent.

Reply:
Easy enough!